### PR TITLE
latency-graph.py: Generates graph for the delete latencies in PG

### PR DESCRIPTION
**Pull request by pdhiran**
This script fetches the top N latency durations from the log file and generates a graph for the same.
Latencies vs Time

Signed-off-by: Pawan Dhiran <pdhiran@redhat.com>

### CODE REVIEW

**Comment by pdhiran** [comment]
Graphs Generated: https://docs.google.com/spreadsheets/d/1YYvB1HZWgh6L2dBW-jmyjrZp1bfAJlCZ0tXdUAtR_NM/edit?usp=sharing

**Comment by pdhiran** [comment]
A standalone script, required for the customer bug : https://bugzilla.redhat.com/show_bug.cgi?id=1952920

**Comment by pdhiran** [comment]
A standalone script, required for the customer bug : https://bugzilla.redhat.com/show_bug.cgi?id=1952920

**Comment by pdhiran** [comment]
A standalone script, required for the customer bug : https://bugzilla.redhat.com/show_bug.cgi?id=1952920

**Review by SrinivasaBharath — APPROVED** [review]
LGTM

File: utility/generate_latency_graph.py [hunk 1]
```diff
@@ -0,0 +1,88 @@
+"""
```
Comment (psathyan) [inline utility/generate_latency_graph.py:1]: Can we move this to `ceph-qe-scripts`?

Comment (pdhiran) [reply utility/generate_latency_graph.py:1]: Created a new PR in ` ceph-qe-scripts ` . https://github.com/red-hat-storage/ceph-qe-scripts/pull/252

**Comment by psathyan** [comment]
@pdhiran can we close this PR?

**Comment by pdhiran** [comment]
merged the Script in ` ceph-qe-scripts `. Closing the PR here.

**Pull request closed without merging**